Doctrine/Orm/OrderFilter.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Dunglas\ApiBundle\Doctrine\Orm;
+
+use Doctrine\ORM\QueryBuilder;
+use Dunglas\ApiBundle\Api\ResourceInterface;
+use Dunglas\ApiBundle\Doctrine\Orm\FilterInterface;
+
+/**
+ * {@inheritdoc}
+ *
+ * @author Théo FIDRY <theo.fidry@gmail.com>
+ */
+class OrderFilter implements FilterInterface
+{
+    /**
+     * @var array
+     */
+    private $blacklist = [];
+
+    /**
+     * @var array
+     */
+    private $whitelist = [];
+
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'order';
+    }
+
+    /**
+     * @param array   $properties List of property names on which the filter will be enabled/disabled.
+     * @param boolean $type       If true properties are passed as a whitelist and as a blacklist otherwise.
```

Comment:
Can you copy the behavior of the new Filter API instead (to preserve consistency): if `$properties` is `null`, can be filtered on all filters, else it's a whitelist (I don't like blacklists because they are error-prone, you always miss to update the list when refactoring).